lib/core/src/Cardano/Wallet/Primitive/AddressDerivation/MintBurn.hs

```diff
@@ -0,0 +1,99 @@
+{-# LANGUAGE DataKinds #-}
+
+-- |
+-- Copyright: © 2018-2021 IOHK
+-- License: Apache-2.0
+--
+-- Derivation of policy keys which are used to create scripts for the purposes
+-- of minting and burning. Derived according to CIP-1855
+-- (https://github.com/cardano-foundation/CIPs/blob/b2e9d02cb9a71ba9e754a432c78197428abf7e4c/CIP-1855/CIP-1855.md).
+--
+-- The policy keys are derived from the following path:
+--
+-- m / purpose' / coin_type' / policy_ix'
+-- m / 1855'    / 1815'      / [2^31 .. 2^32-1]'
+--
+-- Where purpose' and coin_type' are fixed, and each new policy_ix' represents a
+-- different policy key.
+
+module Cardano.Wallet.Primitive.AddressDerivation.MintBurn
+    ( -- * Constants
+      purposeCIP1855
+      -- * Helpers
+    , derivePolicyKey
+    , derivePolicyPrivateKey
+    ) where
+
+import Prelude
+
+import Cardano.Address.Derivation
+    ( XPrv )
+import Cardano.Address.Script
+    ( KeyHash )
+import Cardano.Crypto.Wallet
+    ( deriveXPrv )
+import Cardano.Crypto.Wallet.Types
+    ( DerivationScheme (DerivationScheme2) )
+import Cardano.Wallet.Primitive.AddressDerivation
+    ( Depth (PolicyK, PurposeK, RootK)
```

Comment:
Please just use wildcards for constructor and record imports - then it's less painful to modify the code later.
```suggestion
    ( Depth (..)
```

Reply:
:thinking: I wonder if there's an LSP setting for this. These imports are the way they are because I clicked "LSP: import this for me".

Reply:
Fixed

Reply:
Yes LSP does the same for me, but it's so excessive ... and if you have LSP it shows where a symbol comes from in the tooltip.